docs/product/components/button-groups.html

```diff
@@ -64,30 +64,30 @@
 <section class="stacks-section">
     {% header h2 | Counts %}
     <p class="stacks-copy">
-        Used to display a numerical count inside a button group. Good for large data sets where it's helpful to quickly know how many items are in each filter.
+        Counts are used to display a numerical count inside a button group. This is good for large data sets where it’s helpful to quickly know how many items are behind each filter.
```

Comment:
We should be consistent with our language here. I've been using the same style in various places. We should audit this sometime.